envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -83,8 +83,7 @@ abstract class EnvoyControlTestConfiguration : BaseEnvoyTest() {
         var envoyControls: Int = 1
         var envoys: Int = 1
 
-        // We use envoy version from master. This is 1.14.0-dev.
-        const val defaultEnvoyImage = "envoyproxy/envoy-alpine-dev:5b1723ff54b1a51e104c514ee6363234aaa44366"
+        const val defaultEnvoyImage = "envoyproxy/envoy-alpine:v1.14.3"
```

Comment:
I know that envoy version is hardcoded in at least one more place in this repo :)

Reply:
https://github.com/allegro/envoy-control/blob/master/tools/envoy/Dockerfile

Reply:
@lukidzi can you resolve this?